WalletWasabi/Services/WalletService.cs

```diff
@@ -1025,19 +1025,28 @@ public Operation(Script script, Money amount, string label)
 		/// <param name="allowUnconfirmed">Allow to spend unconfirmed transactions, if necessary.</param>
 		/// <param name="allowedInputs">Only these inputs allowed to be used to build the transaction. The wallet must know the corresponding private keys.</param>
 		/// <param name="subtractFeeFromAmountIndex">If null, fee is substracted from the change. Otherwise it denotes the index in the toSend array.</param>
+		/// <param name="feeTarget">The target number of blocks to estimate the fee.</param>
 		/// <exception cref="ArgumentException"></exception>
 		/// <exception cref="ArgumentNullException"></exception>
 		/// <exception cref="ArgumentOutOfRangeException"></exception>
 		public BuildTransactionResult BuildTransaction(string password,
 														Operation[] toSend,
-														int feeTarget,
+														int? feeTarget = null,
+														Money feeRate = null,
 														bool allowUnconfirmed = false,
 														int? subtractFeeFromAmountIndex = null,
 														Script customChange = null,
 														IEnumerable<TxoRef> allowedInputs = null)
 		{
 			password = password ?? ""; // Correction.
 			toSend = Guard.NotNullOrEmpty(nameof(toSend), toSend);
+
+			if ((feeRate is null && feeTarget is null)
+				|| (feeRate != null && feeTarget != null))
+			{
+				throw new NotSupportedException($"Either {nameof(feeRate)} or {nameof(feeTarget)} must be null. Not both.");
```

Comment:
I think we shouldn't say `Not both.` if `(feeRate != null && feeTarget != null)` is true.